Remotion/Web/Development.WebTesting/WebDriver/Configuration/Chromium/IChromiumConfiguration.cs

```diff
@@ -1,58 +1,59 @@
-// 
-// re-motion is distributed in the hope that it will be useful, 
-// but WITHOUT ANY WARRANTY; without even the implied warranty of 
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
-// GNU Lesser General Public License for more details.
-// 
-// You should have received a copy of the GNU Lesser General Public License
-// along with re-motion; if not, see http://www.gnu.org/licenses.
-// 
-using System;
-using JetBrains.Annotations;
-using OpenQA.Selenium.Chrome;
-
-namespace Remotion.Web.Development.WebTesting.WebDriver.Configuration.Chrome
-{
-  /// <summary>
-  /// Provides configuration specific to initializing and shutting down Chrome.
-  /// </summary>
-  /// <seealso cref="ChromeConfiguration"/>
-  public interface IChromeConfiguration : IBrowserConfiguration
-  {
-    /// <summary>
-    /// Gets the full path to chrome.exe.
-    /// </summary>
-    [NotNull]
-    string BrowserBinaryPath { get; }
-
-    /// <summary>
-    /// Gets the full path to chromedriver.exe.
-    /// </summary>
-    [NotNull]
-    string DriverBinaryPath { get; }
-
-    /// <summary>
-    /// Gets the path to the user directories, which is used by the started Chrome. If the path does not exists, it will be automatically created at startup.
-    /// </summary>
-    [NotNull]
-    string UserDirectoryRoot { get; }
-
-    /// <summary>
-    /// Defines whether the infrastructure should delete the given <see cref="UserDirectoryRoot"/>.
-    /// </summary>
-    bool EnableUserDirectoryRootCleanup { get; }
-
-    /// <summary>
-    /// Creates the <see cref="ChromeOptions"/> used when instantiating the Chrome browser.
-    /// </summary>
-    [NotNull]
-    ExtendedChromeOptions CreateChromeOptions ();
-  }
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// 
+using System;
+using JetBrains.Annotations;
+using OpenQA.Selenium.Chrome;
+using Remotion.Web.Development.WebTesting.WebDriver.Configuration.Chromium.Chrome;
+
+namespace Remotion.Web.Development.WebTesting.WebDriver.Configuration.Chromium
+{
+  /// <summary>
+  /// Provides configuration specific to initializing and shutting down Chrome.
+  /// </summary>
+  /// <seealso cref="ChromeConfiguration"/>
```

Comment:
Is there a file ChromiumConfiguration? If not: 
![image](https://user-images.githubusercontent.com/13847286/65491239-f2c23380-deae-11e9-93b7-98aa8f0bbd43.png)